package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "node ./src/start_server.js",
-    "dev": "nodemon ./src/start_server.js",
+    "dev": "nodemon --ext js,hbs ./src/start_server.js",
```

Comment:
so this makes it restart the server even if we are changing in handlebars too right ?